commands/alpha/rpkg/init/command.go

```diff
@@ -129,3 +128,19 @@ func (r *runner) runE(cmd *cobra.Command, args []string) error {
 	fmt.Fprintf(cmd.OutOrStdout(), "%s created\n", pr.Name)
 	return nil
 }
+
+func (r *runner) packageAlreadyExists(packageName string) error {
+	// only the first package revision can be created from init or clone, so
+	// we need to check that the package doesn't already exist.
+	packageRevisionList := porchapi.PackageRevisionList{}
+	if err := r.client.List(r.ctx, &packageRevisionList, &client.ListOptions{}); err != nil {
```

Comment:
@natasha41575 Does the pacakgeRevision always use `default` namespace? If so, I'm wondering if we shall specify the `namespace` to optimize the request.

Reply:
Besides, since this is client side command, I think we can consider to set a timeout   (e.g. `.List().Timeout(Expiration)`) to avoid long waiting time. Not a blocker for this PR, just something interesting to think about.

Reply:
Thanks for the catch about namespace! packageRevision is namespaced and can be in namespaces other than `default`. I forgot to pass in the namespace, I updated it in the most recent commit. 

Re: setting a timeout, that's an interesting idea. There are a few other client-side commands (like `update`) that also list package revisions so we should set the same timeout. I think we can consider adding that in a separate PR.